src/bp/ui-shared/src/FormFields/TagInputList/index.tsx

```diff
@@ -0,0 +1,83 @@
+import { Position, Tooltip } from '@blueprintjs/core'
+import React, { useEffect, useRef, useState } from 'react'
+import { lang } from '~/translations'
+import { controlKey } from '~/utils/keyboardShortcuts'
+import ShortcutLabel from '~/ShortcutLabel'
+
+import AddButton from '../../Contents/Components/Fields/AddButton'
+
+import { SingleTag } from './SingleTag'
+
+export interface Item {
+  name: string
+  synonyms: string[]
+}
+
+const TagInputList = ({ onChange, placeholder, items, addBtnLabel }) => {
```

Comment:
I think we should add a preventDupes prop and add @Ierezell logic

Reply:
Yeah, I'll add this in a separate PR as a fix, just want to get it merged so we can test everything together